projects/solid/src/index.ts

```diff
@@ -0,0 +1,44 @@
+import {
+    Maskito,
+    MASKITO_DEFAULT_ELEMENT_PREDICATE,
+    type MaskitoElementPredicate,
+    type MaskitoOptions,
+} from '@maskito/core';
+import {
+    createEffect,
+    createResource,
+    createSignal,
+    onCleanup,
+    type Setter,
+} from 'solid-js';
+
+export const useMaskito = ({
```

Comment:
React has the [rule](https://react.dev/learn/reusing-logic-with-custom-hooks#hook-names-always-start-with-use): "Hook names always start with use".

Does `Solid` have the same rule too ?
If no, we should rethink this naming (it should be started with `maskito` word).